data_scrapers/san_francisco_county.py

```diff
@@ -4,101 +4,238 @@
-    return get_json(race_ethnicity_url)
-
-def get_race_json() -> Dict:
-    """ group data by race"""
-    race_query = '?$select=race, sum(confirmed_cases)&$group=race'
-    return get_json(race_ethnicity_url, race_query)
-
-def get_ethnicity_json() -> Dict:
-    """ group data by ethnicity"""
-    ethnicity_query = '?$select=ethnicity, sum(confirmed_cases)&$group=ethnicity'
-    return get_json(race_ethnicity_url, ethnicity_query)
-
-# Confirmed Cases and Deaths by Date and Transmission
-def get_cases_and_deaths() -> Dict:
-    """Get total confirmed cases and deaths by summing over transmission table.
-    Note that total confrimed cases excludes deaths."""
-    query = '?$select=case_disposition, sum(case_count)&$group=case_disposition'
-    return get_json(transmission_url + query)
-
-def get_date_transmission_json() -> Dict:
-    """Get cases by date, transmission, and disposition; order by date ascending."""
-    # date_order_query = '?$order=date'
-    return get_json(transmission_url)
-
-def get_transmission_json() -> Dict:
-    """Group data by transmission category"""
-    transmission_query = '?$select=transmission_category, sum(case_count)&$group=transmission_category'
-    return get_json(transmission_url, transmission_query)
-
-# COVID+ patients from all SF hospitals in ICU vs. acute care, by date
-def get_hospitalization_json() -> Dict:
-    """Get number of COVID+ patients by bed category (ICU or regular), order by date ascending."""
-    date_order_query = '?$order=reportdate'
-    return get_json(hospitalizations_url + date_order_query)
-
-def get_icu_beds() -> Dict:
-    """group data by bed type: ICU or regular"""
-    icu_query = '?$select=dphcategory, sum(patientcount)&$group=dphcategory'
-    return get_json(hospitalizations_url + icu_query)
-
-# Daily count of tests with count and percent of positive tests
-def get_tests() -> Dict:
-    """Get tests by day, order by date ascending"""
-    date_order_query = '?$order=result_date'
-    return get_json(tests_url + date_order_query)
-
-def get_test_totals() -> Dict:
-    """Get total tests and total number of positive tests, to date"""
-    total_test_query = '?$select=sum(tests) as tests'
-    tests = get_json(tests_url + total_test_query)
-    total_positives_query = '?$select=sum(pos) as total_positives'
-    positives = get_json(tests_url + total_positives_query)
-    tests[0].update(positives[0])
-    return tests
+    response = requests.get(race_ethnicity_url)
+    response.raise_for_status()
+    # Dict of target_label : source_label for re-keying.
+    # Note: Native_Amer not currently reported
+    RACE_ETH_KEYS = {"Latinx_or_Hispanic": "Hispanic or Latino", "Asian": "Asian", "African_Amer": "Black or African American",
+                 "White": "White", "Pacific_Islander": "Native Hawaiian or Other Pacific Islander", "Native_Amer": "Native American", "Multiple_Race": "Multiple Race",
+                 "Other": "Other", "Unknown": "Unknown"}
```

Comment:
If you inverted the keys and values here, you wouldn’t need the loop on line 233 and none of your other code would get more complex (you just need to trade `RACE_ETH_KEYS.keys()` for `RACE_ETH_KEYS.values()` on line 207 below).